### ISSUE TITLE
must install git before installing stenv

### ISSUE THREAD

**Issue by fsc137-cfa** [spacetelescope/stenv#78]
Installing stenv on a fresh installation of Ubuntu 22.04, the "conda env create" command failed at "Installing pip dependencies", resulting in the error sequence in this file:
[piperror.txt](https://github.com/spacetelescope/stenv/files/10746251/piperror.txt)

My Linux installation at that point did not have "git" installed, so I did:

sudo apt-get install git-all

and then removed the failed stenv environment:

conda env remove -n stenv

and re-ran the conda enve create command and this time it worked.

**Comment by zacharyburnett** [comment]
Good catch; this appears to be due to `stistools` using `relic` to retrieve the version tag at install-time, by cloning into a git submodule. After https://github.com/spacetelescope/stistools is merged and released in a new `stistools` version, this issue should be resolved.
